src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/templates/category-page/policies/.content.xml

```diff
@@ -15,12 +17,20 @@
                 cq:policy="wcm/foundation/components/responsivegrid/default"
                 jcr:primaryType="nt:unstructured"
                 sling:resourceType="wcm/core/components/policies/mapping">
+                <${confFolderName}
+                    jcr:primaryType="nt:unstructured">
+                    <components
+                        jcr:primaryType="nt:unstructured">
+                        <commerce
+                            jcr:primaryType="nt:unstructured">
+                            <productlist
+                                jcr:primaryType="nt:unstructured"
+                                cq:policy="${confFolderName}/components/commerce/productlist/default"
+                                sling:resourceType="wcm/core/components/policies/mapping"/>
+                        </commerce>
+                    </${confFolderName}>
+                </testing>
```

Comment:
I think there is something wrong, it seems the `components` tag is not properly closed, and that `testing` closing tag is probably wrong here (should be `</${confFolderName}>` ?).

Reply:
Fixed it. The package import seams to be tolerant here and imported the wrong structure in a way the JCR structure was correct.